Example/OCSlimProject_AcceptanceTestsMac/FixtureExample.swift

```diff
@@ -31,5 +32,23 @@ class FixtureExample : NSObject, SlimDecisionTable {
             }
         }
     }
+
+    //MARK: Calling Methods
+
+    func log() -> Bool {
+
+        // Logging Examples
+        print("Swift print() displays simple log statements (OSX only)")
+        NSLog("NSLog() statements include more detailed logging details such as date/time (recommended)")
+
+        // os_log is unsupported for logging and will not be reprted
+        if #available(OSX 10.12, *) {
+            os_log("*WARNING* os_log() statements not captured on OS X")
+        } else {
+            // Fallback on earlier versions
```

Comment:
Need to do something here?

Reply:
Removed the else. It is for example only.